Containers/borgbackup/backupscript.sh

```diff
@@ -134,12 +134,18 @@ if [ "$BORG_MODE" = backup ]; then
     echo "Starting the backup..."
     get_start_time
     if ! borg create "${BORG_OPTS[@]}" "$BORG_BACKUP_DIRECTORY::$CURRENT_DATE-nextcloud-aio" "/nextcloud_aio_volumes/"; then
+        # Add file to Nextcloud container so that it skips any update the next time
+        touch "/nextcloud_aio_volumes/nextcloud_aio_nextcloud_data/skip.update"
+        chmod 777 "/nextcloud_aio_volumes/nextcloud_aio_nextcloud_data/skip.update"
```

Comment:
Todo: move that into start.sh as it could fail elsewhere, too. (Only create if the backupmode is create)

Also: create a fail file that lets any backup fail it the database export fails and remove that of the export is successful

Reply:
done